app/bundles/CoreBundle/Views/GlobalSearch/results.html.twig

```diff
@@ -11,13 +11,15 @@
         </div>
         <div class="panel-body np">
             <ul class="list-group">
-                {% for r in result %}
-                <li class="list-group-item">
-                    {{ r|raw }}
-                </li>
+                {% for k, r in result %}
+                  {% if 'count' is not same as k %}
+                    <li class="list-group-item">
+                        {{ r|raw }}
+                    </li>
+                  {% endif %}
```

Comment:
Found an issue when searching that it would display `count` is specific instances. After looking at the `results.html.php` file, it will unset this in the `$results` array.